opm/parser/eclipse/EclipseState/Schedule/Completion.cpp

```diff
@@ -212,6 +242,22 @@ namespace Opm {
         return m_wellPi;
     }
 
+    int Completion::getSegmentNumber() const {
+        return m_segment_number;
```

Comment:
This should throw an exception if `m_segment_number == -1` -  if calling scope needs to check the (not yet created) `attachedToSegment()` method should be consulted.